jfsd/utils.py

```diff
@@ -375,16 +294,238 @@ def initialize_single_neighborlist(
     neighbor_fn = partition.neighbor_list(displacement,  
                                               jnp.array([Lx, Ly, Lz]),
                                               r_cutoff=space_cut,  # Spatial cutoff for 2 particles to be neighbor
-                                              dr_threshold=0.1,  # displacement of particles threshold to recompute neighbor list
-                                              capacity_multiplier=1,
+                                              #dr_threshold=0.1,  # displacement of particles threshold to recompute neighbor list
+                                              capacity_multiplier=1.1,
                                               format=partition.NeighborListFormat.OrderedSparse,
                                               disable_cell_list=True)
     
     return neighbor_fn
 
+def allocate_nlist(
+    positions: ArrayLike,
+    nbrs: partition.NeighborList) -> partition.NeighborList:    
+    """Allocate particle neighbor list
+    
+    Parameters
+    ----------
+    positions:
+        Array (N,3) of particles positions
+    nbrs:
+        Input neighbor lists
+    
+    Returns
+    -------
+    nbrs
+    
+    """   
+    return nbrs.allocate(positions)
+
+@jit
+def update_nlist(
+    positions: ArrayLike,
+    nbrs: partition.NeighborList) -> partition.NeighborList:    
+    """Update particle neighbor list
+    
+    Parameters
+    ----------
+    positions:
+        Array (N,3) of particles positions
+    nbrs:
+        Input neighbor lists
+    
+    Returns
+    -------
+    nbrs
+    
+    """   
+    #Update neighbor list
+    nbrs = nbrs.update(positions)
+    return nbrs
+
+def CreateHardSphereConfiguration(
```

Comment:
I suppose this completely replaces the random configuration? This is a function, so please use snake_case.

Reply:
changed the name of the function